scripts/exregional_run_prepstart.sh

```diff
@@ -341,6 +341,11 @@ else
       done
     fi
     ncatted -a checksum,,d,, fv_core.res.nc
+
+# generate coupler.res with right date
+    head -1 bk_coupler.res > coupler.res
+    tail -1 bk_coupler.res >> coupler.res
+    tail -1 bk_coupler.res >> coupler.res
```

Comment:
Just to confirm the repeat of line 347 in line 348 is intended, right?

Reply:
Yes, this is intended.